### ISSUE TITLE
Version number in two places

### ISSUE THREAD

**Issue by dentarg** [twingly/twingly-search-api-python#20]
We have it in [`setup.py`](https://github.com/twingly/twingly-search-api-python/blob/5fea52b68d89b0be4fd49f9a6e2da3e7255eb63a/setup.py#L19) and [`twingly_search/__init__.py`](https://github.com/twingly/twingly-search-api-python/blob/5fea52b68d89b0be4fd49f9a6e2da3e7255eb63a/twingly_search/__init__.py#L7) (only two digits here, so maybe we are well aware of it?)

Perhaps in setup.py we can use the version from `__init__.py` [like html5lib does](https://github.com/html5lib/html5lib-python/blob/f796cca5f9ddaaf1e1a8b872f68455551cd3ae2d/setup.py#L36-L46).

**Comment by roback** [comment]
> We have it in setup.py and twingly_search/**init**.py (only two digits here, so maybe we are well aware of it?)

I don't think it was intentional to leave out the last version number here. Its only used in the user agent string. The other clients includes the patch version so I changed it when bumping the version in 5d6d8c52ec7e113b5a93f24849e417e723b0058f.
